srcpkgs/i2pd/template

```diff
@@ -24,7 +24,7 @@ make_dirs="/var/lib/i2pd 0700 _i2pd _i2pd"
 
 case "${XBPS_TARGET_MACHINE}" in
 	x86_64*|i686*) ;;
-	*) make_build_args+=" USE_AESNI=no USE_AVX=no" ;;
+	*) make_build_args+=" USE_AVX=no" ;;
```

Comment:
This only affects archs other than `x86_64` and `i686`. You should instead make a separate case for `i686`. If you test the package you just created, you'll see that you still get that error.

Furthermore, the current template has `USE_AESNI=no`, not `USE_AESNI=1`. And that should be left as it, otherwise it breaks the build for arm targets.

Reply:
I think there're some x86_64 CPU that doesn't speak `AESNI`, too. And there're i686 and armv8-a CPU that speaks AESNI. So, either disable them unconditionally, or make another package for `USE_AESNI=no` is better IMO.
I don't know :-p

Reply:
Or build_options :shrug:

Reply:
One of our ARM crosscompilers didn't understand `-maes`, which the build system tries to use. I don't remember which, unfortunately.